utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,199 @@
+```
+
+or from a local tree with the following command:
+
+```bash
+$ docker build --tag daos-base:centos8 utils/docker/vcluster/daos-base/el8
+```
+
+This Docker file accept the following arguments:
+
+- `RHEL_BASE_IMAGE`: Base docker image to use (default centos)
+- `RHEL_BASE_VERSION`: Version of the base docker image to use (default 8)
+- `BUST_CACHE`: Manage docker building cache (default undefined).  To invalidate the cache, a random
+	value such as the date of the day shall be given.
+- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "no")
+- `DAOS_REPOS`: Space separated list of repos needed to install DAOS (default
+	"https://packages.daos.io/v2.0")
+- `DAOS_GPG_KEYS`: Space separated list of GPG keys associated with DAOS repos (default
+	"https://packages.daos.io/RPM-GPG-KEY")
+- `DAOS_REPOS_NOAUTH`: Space separated list of repos to use without GPG authentication
+	(default "")
+
+For example, building a DAOS base image with authentication enabled could be done with the
+following command:
+
+```bash
+$ docker build --tag daos-base:centos8 --build-arg DAOS_AUTH=yes \
+	https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8
+```
+
+### Building DAOS Nodes Images
+
+The three images `daos-server`, `daos-admin` and `daos-client` could be built directly from GitHub
+or from a local tree in the same way as for the `daos-base` image.  Following command could be used
+to build directly the three images from GitHub:
+
+```bash
+$ for image in daos-server daos-admin daos-client ; do \
+	docker build --tag "$image:centos8" \
+		"https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/$image/el8"; \
+  done
+```
+
+The Docker file of the `daos-server` image accept the following arguments:
+
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "no")
+- `DAOS_HUGEPAGES_NBR`: Number of huge pages to allocate for SPDK (default 4096)
+- `DAOS_IFACE_NAME`: Fabric network interface used by the DAOS engine (default "eth0")
+- `DAOS_SCM_SIZE`: Size in GB of the RAM emulating SCM devices (default 4)
+- `DAOS_BDEV_SIZE`: Size in GB of the file created to emulate NVMe devices (derault 16)
+
+The Dockerfile of the `daos-client` and `daos-admin` images accept the following arguments:
+
+- `DAOS_BASE_IMAGE`: Base docker image to use (default daos-base)
+- `DAOS_BASE_VERSION`: Version of the base docker image to use (default centos8)
+- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "no")
+- `DAOS_ADMIN_USER`: Name or uid of the daos administrattor user (default "root")
+- `DAOS_ADMIN_GROUP`: Name or gid of the daos administrattor group (default "root")
+
+From a local tree, a more straightforward way to build these images could be done with
```

Comment:
Should this be on L94?  Currently you have the following sections here:

1. How to build from github
2. What options are accepted
3. How to build from a checkout

Would it be clearer to have 1, 3, then 2?

Reply:
Indeed, it is better like this: more over this is the layout of the next section.

Reply:
Going to change it.

Reply:
Fixed with commit https://github.com/daos-stack/daos/commit/83667f55edd452cc0e00bfd93c2952c46ef4a951